<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDialog</class>
 <widget class="QDialog" name="InfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_num_points">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of points</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_points">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_addmode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="addmode_value">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_contimode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Conti mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="contimode_value">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_zoommode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="zoommode_value">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_polylines">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Polylines</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_polylines">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_circles">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Circles</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_circles">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
